src/history/hs_verify.c

```diff
@@ -191,7 +191,6 @@ __wt_hs_verify(WT_SESSION_IMPL *session)
         }
 
         WT_ERR(__wt_open_cursor(session, uri_data, NULL, NULL, &ds_cursor));
-        __wt_free(session, uri_data);
```

Comment:
Removing this free is a functional change but it seemed redundant given it is free'd on exit of the function.